conformance/system_macros/add_symbols.ion

```diff
@@ -1,9 +1,85 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Test Cases:
-// add_symbols can be invoked using any type of macro reference.
-// the argument values may be zero or more non-null text values
-// add_symbols does not have any side-effects on the macro table
-// add_symbols should append its arguments to the symbol table of the default module
-// annotations on any argument value should signal an error (because it's not allowed in the symbol table rather than because of any specific validation by the add_symbols macro)
+(ion_1_1 "add_symbols can be invoked"
+         (each "in text with an unqualified macro name"
+               (text ''' (:add_symbols "") ''')
+               "in text with an unqualified macro address"
+               (text ''' (:12 "") ''')
+               "in text with a qualified macro name"
+               (text ''' (:$ion::add_symbols "") ''')
+               "in text with a qualified macro address 12"
+               (text ''' (:$ion::12 "") ''')
+               "in binary with a system macro address 12"
+               (binary "EF 0C 01 90")
+               "in binary with a user macro address"
+               (binary "0C 01 90")
+               (then "and produces only system values"
+                     (produces))
+               (then "and affects the symbol table"
+                     (toplevel '#$1' '#$65' '#$66')
+                     (produces '$ion' 'make_field' ''))))
+
+(ion_1_1 "add_symbols can accept"
```

Comment:
Similarly, I think `add_symbols` and `set_symbols` would also be able to accept `module_name::*`.